.github/ISSUE_TEMPLATE/user-story.md

```diff
@@ -7,29 +7,40 @@ assignees: ''
 
 ---
 
-### Description 
+### Description
 
-<!-- role details / capability details / value details -->
-As <role> I want <capability> so I can <value>
+<!-- For you!
+Please provide a concise description of what this story is about.
+The story should contain the following information:
+    * Who is going to need this? What role can you identify? Try to be as specific as possible. It helps us to understand the need.
```

Comment:
> What role can you identify?

What do you mean by that?

Reply:
What do you mean with what do you mean by that? It is necessary to think about how important this item is so that we can better prioritise the work.

Reply:
I just wanted to point out that `role` is very vague. If you add the sentence that you wrote "think about how important this item is so that we can better prioritise the work." is much more precise and helpful :)